build/pkgs/bliss/spkg-configure.m4

```diff
@@ -0,0 +1,29 @@
+SAGE_SPKG_CONFIGURE([bliss], [
+    m4_pushdef([SAGE_BLISS_MINVER],[0.77])
+    m4_pushdef([SAGE_BLISS_MAJOR],[0])
+    m4_pushdef([SAGE_BLISS_MINOR],[77])
+    SAGE_SPKG_DEPCHECK([cmake], [
```

Comment:
Depcheck on cmake makes no sense unless you use cmake to discover the library (as done in `build/pkgs/onetbb/spkg-configure.m4`)

Reply:
fixed